### ISSUE TITLE
Roadmap: jax.numpy subset

### ISSUE THREAD

**Issue by crutcher** [zspacelabs/bunsen#78]
[jax numpy](https://docs.jax.dev/en/latest/jax.numpy.html) has already identified a semantic subset of numpy which can be expressed over immutable tensors. Many of these functions are implemented in terms of other functions.

As a community roadmap target, lifting this surface to `bunsen` as an incremental project should be a target with high fanout and reward for incremental individual effort.
